.cspell.json

```
@@ -0,0 +1,39 @@
+{
+  "version": "0.2",
+  "gitignoreRoot": ".",
+  "dictionaries": [
+    "css",
+    "html",
+    "fonts",
+    "typescript",
+    "softwareTerms",
+    "companies",
+    "lorem-ipsum",
+    "project-words"
+  ],
+  "dictionaryDefinitions": [
+    {
+      "name": "project-words",
+      "path": "./project-words.txt",
+      "noSuggest": true
+    }
+  ],
+  "ignorePaths": [
+    "**/node_modules/**",
+    "CHANGELOG.md",
+    "examples/**",
+    "packages/docusaurus-theme-translations/locales/**",
+    "**/__tests__/**",
+    "**/package.json",
+    "package.json",
+    "**/yarn.lock",
+    "yarn.lock",
+    "project-words.txt",
+    "website/src/data/users.tsx",
+    "**/*.map",
+    "**/*.xyz",
+    "**/*.docx"
+  ],
+  "ignoreRegExpList": ["Email", "Urls", "{#\\w*}"],
```

```suggestion
  "ignoreRegExpList": ["Email", "Urls", "{#[\\w-]*}"],
```

By my understanding, `\w` doesn't match dashes, right? But anchors can contain dashes?